modules/special-rules/pages/war-engines-in-close-combat.adoc

```diff
@@ -2,40 +2,61 @@
-{blank}[TODO: How do War Engine units -- or a mix of War Engine and regular units -- gang up on on other War Engine units?]
+You can place as many units in base contact with the War Engine unit as its initial Damage Capacity value.
+
+[NOTE.e40k]
+====
+In _Epic 40,000_, you could only 'gang up' with half as many units, which meant that smaller War Engine units like Baneblades could only be attacked by two units -- the same as a regular target!
+====
+//{blank}[TODO: How do War Engine units -- or a mix of War Engine and regular units -- gang up on on other War Engine units?]
+// IJW reply just treat them as units, so that you can actually gang up on similar-size targets.
 
 == Close combat attacks with a War Engine
 
 War engines can also do a lot more damage in close combat than regular units.
 
 * Roll a number of dice to hit equal to half the current Damage Capacity of the War Engine unit (round up).
-* Note that you may allocate hits only among units that the War Engine is actually in base contact with.
-So, even if you score excess hits, your War Engine unit can't destroy more units in close combat than it is actually in base contact with.
+* As normal, you can apply hits to any enemy unit in the combat which is in base contact with any of your units.
 
-{blank}[TODO: In a detachment of multiple War Engine units, can units that are not in base contact with the enemy provide supporting fire like regular units can? If so, is it just +1 to the Assault value total, or should it be '+half the current Damage Capacity' as it is when they support *other* detachments?]
+[NOTE.e40k]
+====
+In _Epic 40,000_, the rules for close combat attacks with War Engines assumed that you had a single War Engine in combat with the enemy, and that there were no other War Engines or allied detachments in the close combat. This revised version treats all the close combat hits normally, instead of limiting the hits to units in base contact with that specific War Engine.
+====
 
 == Close combat attacks against a War Engine
 
 * For each regular unit that you have in base contact with an enemy War Engine, roll a single hit die as normal.
 * Ignore power fields and void shields for any hits that you score in close combat -- they damage the War Engine directly.
 Roll for critical damage as normal.
 
+== War Engine detachments in close combat
+
+* When a War Engine detachment is in close combat and some of the War Engine units are not in base contact with the enemy, do not add +1 for the units not in base contact.
+Instead, add a number equal to half the current Damage Capacity of these War Engines.
+This does not prevent the War Engine units from supporting other close combats.
+
+[NOTE.e40k]
+====
+In _Epic 40,000_, there were rules for War Engines to support *other* close combats, but no rules for supporting a close combat that the War Engine detachment was in. We have adapted the rules for supporting other close combats.
+====
+
```

Comment:
Some tweaks — please check that I don't 'break' anything.
```suggestion
== War Engine detachments in close combat

* When you add up Assault values for a War Engine detachment that is in close combat, do not add +1 for 'supporting fire' from each unit in the detachment that is within 15 cm of the enemy but not in base contact.
Instead, add a number equal to half the current Damage Capacity of each such War Engine unit.
+
To be clear, such units may still also support *other detachments* as we explain at xref:war-engines-in-support-of-close-combats.adoc[].

[NOTE.e40k]
====
In _Epic 40,000_, the rules specify how War Engine units can support other detachments that are in close combat, but not how they can provide supporting fire within their own detachment.
We make both situations plain.
====

```

Reply:
This appears to contradict other statements and is thus 'wrong': `This does not prevent the War Engine units from supporting other close combats.`

The original rules state `In order to lend its support to a close combat a war engine must be within 15cm of an enemy unit which is fighting in, or lending support to, the combat and the war engine can't be in close combat itself.` ...and our current Remastered rules mirror this with `To do this, your War Engine unit must not be engaged in combat itself`.

So, I'll tweak my suggestion here a bit further, thus: (along with some other further small tweaks while I'm at it)

```suggestion
== War Engine detachments in close combat

When you add up Assault values for a War Engine detachment that is in close combat, do not add +1 for 'supporting fire' from each unit in the detachment that is within 15 cm of the enemy but not in base contact.
Instead, add a number equal to half the current Damage Capacity of each such War Engine unit.

[NOTE.e40k]
.Supporting fire from War Engine units 
====
_Epic 40,000_ gives rules for War Engine units in support of other detachments that are in close combat -- but does not gives rules for how they can provide supporting fire within their own detachment.
We make both situations plain.
====

```

Reply:
> This appears to contradict other statements and is thus 'wrong': `This does not prevent the War Engine units from supporting other close combats.`

Could you explain in a bit more detail? I can't see where this is contradicting other statements. As a reader, I no longer know how this affects War Engines supporting other close combats.

Reply:
The detail I already added above covers it, doesn't it?

> The original rules state `In order to lend its support to a close combat a war engine must be within 15cm of an enemy unit which is fighting in, or lending support to, the combat and the war engine can't be in close combat itself.` ...and our current Remastered rules mirror this with `To do this, your War Engine unit must not be engaged in combat itself`.

...where original is [War engines supporting close combats](https://thehobby.zone/resources/e40k-compendium/Content/SpecialRules/WarEngines/WarEnginesSupportingCloseCombats.htm) and Remastered is [War engines in support of close combats](https://epicremastered.com/core/special-rules/war-engines-in-support-of-close-combats.html).

Note "can't be in close combat itself" and "must not be engaged in combat itself", respectively. Unless I'm missing something daft (always possible!) these seem to directly contradict that particular line that I've excluded from my suggestion.

Reply:
I think the missing bit is that close combats are resolved sequentially rather than simultaneously.

So a War Engine detachment can be in a close combat (with one of them outside base contact), resolve their own close combat, and then be in a position to support another separate close combat that hasn't been resolved yet. 

Without the removed bullet, it's unclear whether the 'supporting' War Engine can support its own detachment and then support the separate close combat.

Reply:
Ah, okay. So, resolve all close combats, but choose the order carefully to maximise support from war engines... don't treat close combats as simultaneous.

I made an implicit assumption that although of course we resolve them one at a time, we nonetheless treat them as simultaneous. As we are **not** supposed to do that, **should we make this more clear somewhere**? (Others might make the same assumption that I did.) Or is it already clear somewhere and I've just overlooked/forgotten statements to that effect?

Latest iteration of the suggestion:

```suggestion
== War Engine detachments in close combat

When you add up Assault values for a War Engine detachment that is in close combat, do not add +1 for 'supporting fire' from each unit in the detachment that is within 15 cm of the enemy but not in base contact.
Instead, add a number equal to half the current Damage Capacity of each such War Engine unit.

To be clear, such units may still also support *other detachments* as we explain at xref:war-engines-in-support-of-close-combats.adoc[].

[NOTE.e40k]
.Supporting fire from War Engine units 
====
_Epic 40,000_ gives rules for War Engine units in support of other detachments that are in close combat -- but does not give similar rules to apply the same logic for supporting fire within their own detachment (so only the usual +1 would apply, which seems odd).
We make both situations plain.
====
```

Reply:
@IJW-Wartrader Any thoughts on my last comment on this thread? If not then I'll just apply this suggestion and proceed to final checks with a view to publish, as there are no other outstanding items.

Reply:
@digi-brain that looks OK.

> I made an implicit assumption that although of course we resolve them one at a time, we nonetheless treat them as simultaneous. As we are **not** supposed to do that, **should we make this more clear somewhere**? (Others might make the same assumption that I did.) Or is it already clear somewhere and I've just overlooked/forgotten statements to that effect?

Hmm. Not sure. For me, 'The player with the Initiative decides the order in which to resolve close combats.' from close-combat-procedure.adoc makes it clear that you resolve one CC, then move onto the next and resolve that, etc.

Reply:
> Well, my point is that **of course** you must 'decide the order' to resolve one at a time, because that is the **only** practical way to do it in a tabletop wargame. So, by itself, this:
> 
> > The player with the Initiative decides the order in which to resolve close combats.
> > ...
> > To resolve a close combat, use the close combat procedure as follows.
> 
> ...doesn't preclude the possibility that we treat the set of combats that a player triggers as simultaneous events.

If you think it's unclear, then sure, add something. But I may not have explained myself properly.

<img width="745" alt="Screen Shot 2022-01-04 at 16 01 54" src="https://user-images.githubusercontent.com/17686203/148087513-ca870b97-d5e9-4c20-b1ee-289d047006d6.png">

You choose the order in which to resolve the combats (first red underline). Then you resolve **a combat** (second red underline), following the six steps (green bar). This includes one of the detachments losing and making a Retreat move, physically moving the models. Then you go to the next combat and repeat the six steps. Everything that happened in the first combat has been resolved and can potentially affect the second combat. In particular, where the losers retreated to and how this affects any 'new' Retreat moves as there may no longer be enough space to retreat into. 

I can't see a way to read the current rules text in a different way. For example, Retreat moves would have to be totally rewritten so that you check the 15 cm distance at the end of the phase, instead of the end of the Retreat move.